scripts/test-remote-dev-context7-adoption.py

```
@@ -0,0 +1,163 @@
+
+
+def run_manager(codex_home: Path, arguments: list[str], *, input_text: str | None = None) -> str:
+    result = subprocess.run(
+        [sys.executable, str(MANAGER), *arguments],
+        input=input_text,
+        text=True,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        env=manager_environment(codex_home),
+        check=False,
+    )
```

Ruff (0.16.1)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)